lib/rules/web/admin_console/4.4/monitoring_alerts.xyaml

```diff
@@ -0,0 +1,112 @@
+click_icon_index_button:
+  element:
+    selector:
+      xpath: (//button[@aria-label='<aria_label_text>'])[<index>]
+    op: click
+set_placeholder_input_value:
+  element:
+    selector:
+      xpath: (//input[@placeholder='<placeholder_text>'])[<index>]
+    op: send_keys <input_value>
+    type: input
+clear_label_input_value:
+  elements:
+  - selector:
+      xpath: //label[text()='<label_text>']/..//input
+    op: clear
+    type: input
+set_label_input_value:
+  element:
+    selector:
+      xpath: //label[text()='<label_text>']/..//input
+    op: send_keys <input_value>
+    type: input
+check_div_text:
+  element:
+    selector:
+      xpath: //div[contains(text(),'<text>')]
+
+open_alert_detail:
+  action: click_alert_link_with_text
+silence_alert_from_action:
+  action: click_actions_button
+  action: click_silence_alert_button
+  action: click_create_button
+
+click_alert_link_with_text:
```

Comment:
This can be defined as a general rule with name `click_link_with_text`
```
xpath: //a[text()='<text>']
```

Reply:
didn't change, for the alert here need more time, default 10 seconds is not enough